.github/workflows/copilot-build-memorypipeline.yml

```diff
@@ -0,0 +1,72 @@
+﻿name: memorypipeline-build
```

Comment:
memoryservice-build would probably be a better name

Reply:
Change to `copilot-build-memorypipeline` for consistency.